boards/arm/nucleo_h745zi_q/nucleo_h745zi_q.dtsi

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <st/h7/stm32h743zitx-pinctrl.dtsi>
```

Comment:
The pinctrl file does not point to the 745 SoC.

Reply:
You're absolutely right. It is fixed now (see last push). I have tested it and `hello_world` sample still works properly.